nubia/internal/cmdbase.py

```diff
@@ -409,6 +409,10 @@ def _find_subcommand_attr(self, name):
         for attr, inspection in subcommands:
             if inspection.command.name == name:
                 return attr
+            else:
```

Comment:
It would be nice to simplify this by doing:
```python
if inspection.command.name == name or name in inspection.command.aliases:
    return attr
```

Reply:
Done!